### DISCUSSION TITLE
Is there a way to explicitly set/override the github discussion i want associated with a given page/note?

### DISCUSSION THREAD

**Discussion by rufuspollock** [flowershow/flowershow#459] [General and Help] [answered]
My use case:
- i've create a discussion in github discucssions.
- There were comments.
- Later on i migrated main content of that discussion into my digital garden.
- I want to now have those comments show up on my post in the digital garden ... (ie. to associate that github discussion with that note?)

Question: Is there a way to explicitly set/override the github discussion i want associated with a given page/note in my website / digital garden?

e.g. in the frontmatter i'd have something like:

```md
giscus_id: 270
```

And then the discussion from discussion 270 would show up on that note.

**Comment by rufuspollock** [comment]
@khalilcodes as implementer of the discussions stuff perhaps you could answer this one ...

**Comment by khalilcodes** [comment, marked as answer]
@rufuspollock yes this is possible to do by checking explicitly if a frontmatter field `giscus_id` exists and then use that for mapping else default to using the pathname. The `giscus_id` would be the discussion number.

<img width="680" alt="Screen Shot 2023-04-11" src="https://user-images.githubusercontent.com/42637597/231168676-4d1ef844-7286-430b-957d-bd9e7db91624.png">

Another alternative can be that you can modify the title of the discussion to have the new page's pathname and giscus should map that automatically, since that is the mapping method setup currently.

<img width="583" alt="Screen Shot 2023-04-11" src="https://user-images.githubusercontent.com/42637597/231162891-6f86fdc5-fa99-4236-a853-dfb1bc747f41.png">

**Reply by rufuspollock** [reply]
@khalilcodes ok and what is the `giscus_id`. My use case: i have an existing discussion i want to association to a given note - what would be `giscus_id` for that .

**Reply by khalilcodes** [reply]
@rufuspollock That would be the discussion number.

<img width="680" alt="Screen Shot 2023-04-11" src="https://user-images.githubusercontent.com/42637597/231168676-4d1ef844-7286-430b-957d-bd9e7db91624.png">

**Reply by rufuspollock** [reply]
@khalilcodes to clarity: does this functionality already exist and can you point to the main docs for it (or can we update the docs so it is clear) 🙏

**Reply by khalilcodes** [reply]
@rufuspollock sorry I meant this is possible to implement as a feat, and does not yet exist in current flowershow.

**Reply by rufuspollock** [reply]
@khalilcodes so there is no way to do this currently, correct?

And how does current system work? and where is that documented?

**Reply by khalilcodes** [reply]
> so there is no way to do this currently, correct?

@rufuspollock Yes correct.

Some explanation on working with additional links are in docs here - https://flowershow.app/docs/comments#developers
